src/app/AccountDetails/AccountDetails.tsx

```diff
@@ -0,0 +1,295 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from 'react-router-dom';
+import {
+  PageSection,
+  PageSectionVariants,
+  Tabs,
+  Tab,
+  TabContent,
+  TabContentBody,
+  TabTitleText,
+  Title,
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+  Label,
+  Flex,
+  FlexItem,
+  Page,
+  Spinner,
+  LabelGroup,
+} from "@patternfly/react-core";
+import InfoCircleIcon from "@patternfly/react-icons/dist/js/icons/info-circle-icon";
+import { Table, Tbody, Td, Th, Thead, Tr } from "@patternfly/react-table";
+import { getAccountByName, getAccountClusters } from "../services/api";
+import { Link  } from "react-router-dom";
+import { AccountData, Cluster } from "@app/types/types";
+import { useLocation  } from "react-router-dom";
+interface LabelGroupOverflowProps {
+  labels: {
+    text: string;
+  }[];
+}
+
+const LabelGroupOverflow: React.FunctionComponent<LabelGroupOverflowProps> = ({
+  labels,
+}) => (
+  <LabelGroup>
+    {labels.map((label, index) => (
+      <Label key={index}>{label.text}</Label>
+    ))}
+  </LabelGroup>
+);
+
+const AggregateClustersPerAccount: React.FunctionComponent = () => {
+  const [data, setData] = useState<Cluster[] | []>([]);
+  const [loading, setLoading] = useState(true);
+  const { accountName } = useParams();
+
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+          console.log("Fetching data...");
+          const fetchedAccountClusters = await getAccountClusters(accountName);
+          console.log("Fetched Account data:", fetchedAccountClusters);
+          setData(fetchedAccountClusters);
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchData();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  console.log("Rendered with data:", data);
+
+  return (
+    <React.Fragment>
+      {loading ? (
+        <div
+          style={{
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            height: "100vh",
+          }}
+        >
+          <Spinner size="xl" />
+        </div>
+      ) : (
+        <Table aria-label="Simple table">
```

Comment:
When I click on the sidebar menu`Accounts` - `<account_name>` - `Clusters` tab the layout is broken
![Screenshot from 2023-11-28 09-52-34 - 29771932](https://github.com/RHEcosystemAppEng/cluster-iq-console/assets/89217163/b40dd188-3f3e-4900-9399-6df8493e57c9)

Reply:
Fixed in this PR #15 
![image](https://github.com/RHEcosystemAppEng/cluster-iq-console/assets/51123977/aecbb41f-085c-4717-8ed1-217868116d24)